requirements-dev.txt

```diff
@@ -10,4 +10,5 @@ botocore==1.6.4
 multidict==3.1.3
 wrapt==1.10.11
 dill==0.2.7.1
-packaging==16.8
\ No newline at end of file
+packaging==16.8
+-e .
```

Comment:
I think -e should be first, as I believe the last requirement should work.  In fact the tests should be failing because my patch test will fail with botocore==1.6.4.  I mentioned this in jettify's version as well.

Reply:
I would agree about the tests but both tests ran with botocore at 1.5.78, and 1cc8450 only Py3.5 failed on S3 tests, and after triggering Travis again d72276d then Py3.6 failed on Dynamo and SQS. Which to me indicated somethings wrong with the starting of moto/the environment/the box, or timing tests, otherwise I would expect the failing tests to be the same, and the same version of python.

Yep you are correct, `-e .` should be first.